tg/models/__init__.py

```diff
@@ -213,7 +224,33 @@ def get_msgs(self, chat_id, offset=0, limit=10):
         # if len(self.msgs[chat_id]) >= offset + limit:
         #     break
 
-        return sorted(self.msgs[chat_id], key=lambda d: d['id'])[::-1][offset:limit]
+        return sorted(self.msgs[chat_id], key=lambda d: d["id"])[::-1][
+            offset:limit
+        ]
+
+    def send_message(self, chat_id, text):
+        log.info('Sending msg')
+        result = self.tg.send_message(chat_id=chat_id, text=text)
+
+        result.wait()
+        if result.error:
+            log.info(f'send message error: {result.error_info}')
+        else:
+            log.info(f'message has been sent: {result.update}')
+
+    def delete_msg(self, chat_id):
+        if chat_id is None:
+            return False
+        selected_msg = self.current_msgs[chat_id]
+        msg_item = self.msgs[chat_id].pop(selected_msg)
+        self.current_msgs[chat_id] = min(
+            selected_msg, len(self.msgs[chat_id]) - 1
+        )
+        log.info(f"Deleting msg from the chat {chat_id}: {msg_item}")
+        message_ids = [msg_item["id"]]
+        r = self.tg.delete_messages(chat_id, message_ids, revoke=True)
```

Comment:
Maybe we could try to delete msg first and if success delete locally?

Reply:
Yep, we could try to listen for `updateDeleteMessages`. So, we could use it for messages removal. 
But first, let's merge https://github.com/paul-nameless/tg/pull/11/files#diff-0b1338b3fa65880d711319687a8f2a45R204 into master, so `remove_message`  function will be available in the codebase

Reply:
So, let's handle it in the next PR